products/suse-manager.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SUSE Manager
+category: server-app
+tags: suse
+iconSlug: suse
+permalink: /suse-manager
+versionCommand: zypper info SUSE-Manager-Server-release
+releasePolicyLink: https://www.suse.com/lifecycle
+changelogTemplate: "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/{{'__RELEASE_CYCLE__'}}/"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: General Support
+
+releases:
+-   releaseCycle: "4.3"
+    releaseDate: 2022-06-20
+    eol: 2025-06-30
+    latest: '4.3.11.1'
```

Comment:
Looks like some latest releases must be updated.

Reply:
Good one, 4.3.13 was missing - I fixed it with commit 818563d89e486b9ceb394c3a54c5893adae53e22. What do you think?